.github/workflows/test.yaml

```diff
@@ -178,7 +178,9 @@ jobs:
               ;;
               wasmedge-build)
                   sudo docker build -t wasmedge tests/wasmedge-build
-                  sudo docker run --privileged --cgroupns=host --rm -v /sys/fs/cgroup:/sys/fs/cgroup:rw,rslave  -w /crun -v ${PWD}:/crun wasmedge
```

Comment:
I am confused what is going on here.  You are destroying the hosts /var/lib/containers directories and then running docker to use them.  I am guessing Podman is running within the docker container.  one question I have is why not just setup a tmpfs for /var/lib/containers within the docker conainer and then remove it when you are done with it, rather then destroying the hosts /var/lib/containers dir?

Reply:
I am just using it as a volume.  I agree, it is confusing.  I've dropped this part and replaced it with a regular managed volume